docs/source/whatsnew.rst

```diff
@@ -138,6 +135,33 @@ To learn more about the new private data patterns, see :doc:`private-data/privat
 for information about private data sharing and :doc:`private-data-arch` (reference information)
 for documentation about private data collection configuration and implicit collections.
```

Comment:
Is this section all one sentence? If so there needs to be commas. Otherwise I think we need periods after each for clause and capitalize the `f`.

Reply:
I think you have to hit the expansion pain icon on the left-hand side to expose the text that occurs in the doc prior to the lower-case 'f' you are enquiring about.

Reply:
refectoring this sentence.